k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java

```diff
@@ -149,6 +155,12 @@ public void onTokenChanged(Recipient recipient) {
         });
     }
 
+    @Override
+    public void beforeDragStart() {
+        View recipientExpander = activity.findViewById(R.id.recipient_expander);
+        recipientExpander.performClick();
```

Comment:
this logic should be in RecipientPresenter. see `setPresenter` for how this is handled in other cases, it should be simple to forward the onDrag events to the presenter, and handle visibility changes from there.